Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 2
Rules claiming more threads will be scaled down.
Job stats:
job               count
--------------  -------
all                   1
call                  2
merge_coverage        1
merge_freq            1
total                 5

Select jobs to execute...

[Thu Aug 29 14:17:56 2024]
rule call:
    input: data/reference_index/REJOc-reference.fa, data/sorted_reads/CD00-m000-00-REJOc.bam, data/sorted_reads/CD00-m000-00-REJOc.bam.bai, data/bed_file/REJOc.bed
    output: data/calls/CD00-m000-00-REJOc.csv, data/coverage/CD00-m000-00-REJOc.csv
    jobid: 11
    reason: Set of input files has changed since last execution
    wildcards: sample=CD00-m000-00-REJOc
    resources: tmpdir=/var/folders/k0/3b7hrv0d2n7fs2g57vkng1fm0000gp/T

[Thu Aug 29 14:17:56 2024]
rule call:
    input: data/reference_index/JRCSF-reference.fa, data/sorted_reads/CD00-m000-00-JRCSF.bam, data/sorted_reads/CD00-m000-00-JRCSF.bam.bai, data/bed_file/JRCSF.bed
    output: data/calls/CD00-m000-00-JRCSF.csv, data/coverage/CD00-m000-00-JRCSF.csv
    jobid: 10
    reason: Set of input files has changed since last execution
    wildcards: sample=CD00-m000-00-JRCSF
    resources: tmpdir=/var/folders/k0/3b7hrv0d2n7fs2g57vkng1fm0000gp/T

[Thu Aug 29 14:17:57 2024]
Error in rule call:
    jobid: 10
    input: data/reference_index/JRCSF-reference.fa, data/sorted_reads/CD00-m000-00-JRCSF.bam, data/sorted_reads/CD00-m000-00-JRCSF.bam.bai, data/bed_file/JRCSF.bed
    output: data/calls/CD00-m000-00-JRCSF.csv, data/coverage/CD00-m000-00-JRCSF.csv

[Thu Aug 29 14:17:57 2024]
Error in rule call:
    jobid: 11
    input: data/reference_index/REJOc-reference.fa, data/sorted_reads/CD00-m000-00-REJOc.bam, data/sorted_reads/CD00-m000-00-REJOc.bam.bai, data/bed_file/REJOc.bed
    output: data/calls/CD00-m000-00-REJOc.csv, data/coverage/CD00-m000-00-REJOc.csv

RuleException:
CalledProcessError in file /Users/anitido/Documents/GitHub/Escapability/Viral Escape Analysis/Snakefile, line 105:
Command 'set -euo pipefail;  /Users/anitido/miniforge3/envs/viral_escape/bin/python3.10 '/Users/anitido/Documents/GitHub/Escapability/Viral Escape Analysis/.snakemake/scripts/tmplz5_0ncy.codon_caller.py'' returned non-zero exit status 1.
  File "/Users/anitido/Documents/GitHub/Escapability/Viral Escape Analysis/Snakefile", line 105, in __rule_call
  File "/Users/anitido/miniforge3/envs/viral_escape/lib/python3.10/concurrent/futures/thread.py", line 58, in run
RuleException:
CalledProcessError in file /Users/anitido/Documents/GitHub/Escapability/Viral Escape Analysis/Snakefile, line 105:
Command 'set -euo pipefail;  /Users/anitido/miniforge3/envs/viral_escape/bin/python3.10 '/Users/anitido/Documents/GitHub/Escapability/Viral Escape Analysis/.snakemake/scripts/tmpw7qsefby.codon_caller.py'' returned non-zero exit status 1.
  File "/Users/anitido/Documents/GitHub/Escapability/Viral Escape Analysis/Snakefile", line 105, in __rule_call
  File "/Users/anitido/miniforge3/envs/viral_escape/lib/python3.10/concurrent/futures/thread.py", line 58, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-08-29T141755.870563.snakemake.log
